ledger/catchpointtracker.go

```diff
@@ -364,6 +379,12 @@ func (ct *catchpointTracker) newBlock(blk bookkeeping.Block, delta ledgercore.St
 		}
 		ct.accountDataResourceSeparationRound = blk.BlockHeader.Round + basics.Round(catchpointLookback)
 	}
+
+	if config.Consensus[blk.CurrentProtocol].EnableOnlineAccountCatchpoints {
+		atomic.StoreInt32(&ct.reenableGeneratingCatchpointFiles, 1)
+	} else {
+		atomic.StoreInt32(&ct.reenableGeneratingCatchpointFiles, 0)
```

Comment:
I think we do not need resetting it, it is not likely we'll change `EnableOnlineAccountCatchpoints` value